### PR TITLE
Offer go back to classic when switched to lens

### PR DESCRIPTION
**Pull request by codefromthecrypt**
In a previous change, we offered to switch to Lens. This offers to
switch back, based on the same cookie (`lens`). This should help
reduce risk of people not wanting to deploy.

Note: this needs css work and also I'm not sure why the page isn't
reloading completely. I would like some help finishing this.

### CODE REVIEW

**Comment by codefromthecrypt** [comment]
cc @zeagord @tacigar can one of you help finish this?

File: zipkin-lens/src/components/App/Sidebar.js [hunk 1]
```diff
@@ -24,6 +25,14 @@ const pageInfo = {
 };
 
 class Sidebar extends React.Component {
+  // Before Zipkin 2.13, Lens was optionally available based on a cookie named 'lens'.
+  // To revert to the classic UI, remove the cookie and reload.
+  goBackToClassic(evt) {
+    evt.preventDefault();
+    Cookies.remove('lens');
+    this.window.location.reload(true);
```
Comment (codefromthecrypt) [inline zipkin-lens/src/components/App/Sidebar.js]: the cookie is removed, but the page isn't reloaded...

Comment (tacigar) [reply zipkin-lens/src/components/App/Sidebar.js]: `this.` is not needed.

Comment (tacigar) [reply zipkin-lens/src/components/App/Sidebar.js]: OK. I'll fix it.

Comment (codefromthecrypt) [reply zipkin-lens/src/components/App/Sidebar.js]: lint made me put this there.. and yeah maybe that's what broke it :P

File: zipkin-ui/templates/layout.mustache [hunk 2]
```diff
@@ -7,7 +7,7 @@
           <ul class="navbar-nav mr-auto">
             <li class="nav-item" data-route="index"><a class="nav-link"  href="{{contextRoot}}" data-i18n="nav.find">Find a trace</a></li>
             <li class="nav-item" data-route="traceViewer"><a class="nav-link"  href="{{contextRoot}}traceViewer" data-i18n="nav.viewSaved">View Saved Trace</a></li>
-            <li class="nav-item" data-route="dependency"><a class="nav-link"  href="{{contextRoot}}dependency/" data-i18n="nav.dep">Dependencies</a></li>
```
Comment (tacigar) [inline zipkin-ui/templates/layout.mustache:10]: This trailing slash causes redirecting issue in dependency page.

File: zipkin-lens/src/components/App/Sidebar.js [hunk 3]
```diff
@@ -24,6 +29,24 @@ const pageInfo = {
...
+  constructor(props) {
+    super(props);
+    this.goBackToClassic = this.goBackToClassic.bind(this);
+  }
+
+  goBackToClassic(event) {
+    const { location, history } = this.props;
+
+    Cookies.remove('lens');
+    if (location.pathname === '/zipkin') {
```
Comment (tacigar) [inline zipkin-lens/src/components/App/Sidebar.js:41]: Zipkin-ui requires trailing slash in root page (`/zipkin`)
so if location is `/zipkin`, add trailing slash

https://github.com/openzipkin/zipkin/blob/649aeefdbaf68be7951c44ac0f689acd10b24a03/zipkin-ui/js/main.js#L23
https://github.com/openzipkin/zipkin/blob/649aeefdbaf68be7951c44ac0f689acd10b24a03/zipkin-ui/js/publicPath.js#L9-L11

**Comment by tacigar** [comment]
gobacktolens

**Comment by codefromthecrypt** [comment]
I clicked from every screen and am very very happy!

**Pull request merged**